platforms/hyperledger-fabric/configuration/roles/create/new_organization/orderer_org/appchannel/templates/update_channel_script.tpl

```diff
@@ -3,9 +3,16 @@
 set -x
 
 CURRENT_DIR=${PWD}
+NETWORK_VERSION="{{ version }}"
 
-echo "installing jq "
-apt-get install -y jq
+if [ "$NETWORK_VERSION" != "2.5.4" ]; then
+    echo "installing jq "
```

Comment:
This should use the package_manager script

Reply:
Done